### DISCUSSION TITLE
the program does not install correctly? cannot locate MTGA_Draft_Tool.exe

### DISCUSSION THREAD

**Discussion by Iksander21** [unrealities/MTGA_Draft_17Lands#46] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/7c273081-3a19-4302-be6f-06bb1db7a4c6)
Hi, i have tried to install the program but, as show in in the screen shot, only a few of the files appear in the installation folder.
I have tried both as an administrator and moving the installation folder to different drives, but nothing seem to change.

I am using an Acer Nitro AN517-52 laptop with window 10 home edition

Any help would be appreciated.
Thank You

**Comment by FiYir** [comment]
@Iksander21 Where did you install the app? Was it the root directory (`C:` or `D:`)?

If you didn't install it to either of those drives, you could try uninstalling it by clicking `unins000.exe`, and then reinstalling it to the root directory of your drive.

![image](https://github.com/user-attachments/assets/3d487fee-d3ca-4fa6-a4d7-b4094b95d21a)

**Reply by Iksander21** [reply]
I have been using the default file path.
C:\MtgaDraftTool

**Reply by FiYir** [reply, marked as answer]
@Iksander21 Did Windows Defender quarantine the exe? 

If you go to `Virus & threat protection>Protection history` what do you see?

![image](https://github.com/user-attachments/assets/f7305d79-a03f-4062-af03-287bd04e52b6)

**Reply by Iksander21** [reply]
It was AVG quarantining the MTGA_Draft_Tool.exe file.

Thank you.
